src/riak_kv_entropy_manager.erl

```diff
@@ -79,6 +80,7 @@
 
 -define(DEFAULT_CONCURRENCY, 2).
 -define(DEFAULT_BUILD_LIMIT, {1, 3600000}). %% Once per hour
+-define(KV_ENTROPY_LOCK_TIMEOUT, app_helper:get_env(riak_kv, anti_entropy_lock_timeout, 10000)).
```

Comment:
same idea mentioned in https://github.com/basho/yokozuna/pull/644#discussion_r64286497. `10000` may be nicer as it's own default macro.